psi4/src/read_options.cc

```diff
@@ -1439,6 +1439,14 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- When using |scf__stability_analysis| ``FOLLOW``, maximum number of orbital optimization attempts
             to make the wavefunction stable. !expert -*/
         options.add_int("MAX_ATTEMPTS", 1);
+        /*- Do Perform Incremental Fock Build? -*/
+        options.add_bool("IFB", true);
+        /*- Do Perform Density Screening for Direct SCF? [Haser:1989:104] -*/
+        options.add_bool("SCF_DENSITY_SCREENING", true);
+        /*- Threshold for SCF Density Screening -*/
+        options.add_double("DENSITY_SCREENING_TOLERANCE", 1.0e-13);
+        /*- Density Threshold to stop Incremental Fock Build -*/
+        options.add_double("IFB_D_CONVERGENCE", 1.0e-5);
```

Comment:
This could just be `INCFOCK_CONVERGENCE`

Reply:
Susi has a good point: "IFB" is too opaque. Please rename the option.